vault/dendron.topic.traits.quickstart.md

```diff
@@ -0,0 +1,90 @@
+---
+id: EQoaBI8A0ZcswKQC3UMpO
+title: Quickstart
+desc: ''
+updated: 1638860823247
+created: 1638843110902
+---
+
+This example will show you how you can create your own note traits and apply them to your new notes.  In this example, we'll create a trait that will modify the name and title of the note automatically when you create a new note.
+
+> ❗️ Note: This is an experimental feature that may have breaking changes in future iterations.
```

Comment:
add note reference to `#stage.germ`

Reply:
I'll be making this change within the changelog update for early seed

Reply:
See https://github.com/dendronhq/dendron-site/pull/317